test/deep-equals.specs.ts

```diff
@@ -0,0 +1,16 @@
+import 'mocha'
+import fc from 'fast-check'
+import { deepEquals } from '../src/deep-equals'
+import { twoObjectsEqualsInContent, twoObjectsInequalsInContent, valueAndTargetSameKeysDifferentValues } from './generators'
+
+describe('deepEquals', () => {
+	it('should return true when two objects are equals in content order insensitive', () => {
```

Comment:
indent is still wrong, that's getting a bit long :/

Reply:
Done